source/testnet/guides/multisig-transfer.rst

```diff
@@ -0,0 +1,175 @@
+
+#.  Select the relevant recipient from the list. If there are many recipients in the list, you can use search to find the right recipient. Select **Continue**. You can now set up a release schedule.
+
+Set the **Transaction expiry** time. You must submit proposals to the chain within the last 2 hours up to the expiry date. This means you must set the expiry time so that it’s possible for any co-signers to return their signatures in time. Select **Continue**.
+
+Add a release schedule
+======================
+
+You’re now ready to create a release schedule where you can specify when you want the transfer to take place. You can choose between a regular interval schedule and an explicit schedule.
+
+Option 1: Create a regular interval schedule
+--------------------------------------------
+
+#.  Select **Regular interval** in **Schedule type**.
+
+#.  Select the frequency with which you want the transfers to be released in **Release every**.
+
+#.  Specify how many intervals you want to split the transfer into in **Split transfer in**.
+
+#.  Specify the date and time you want the first transfer to take place in **Starting**.
+
+#.  When the schedule is complete, select **Continue**. You can now approve the transaction on the Ledger. You can see the release schedule under **Transaction Details.**
+
+.. Note::
+   You can divide a transfer into a maximum of 255 releases.
+
+Option 2: Create an explicit schedule
+-------------------------------------
+
+#. Select **Explicit schedule**. For each release you want in the schedule, you have to specify the amount and the release time.
+
+#. Select **Add release to schedule** in **Releases**.
+
+#. Enter the amount you want to transfer in this release in **Amount**.
+
+#. Specify the date and time you want the transfer to take place in **Release time**.
+
+#. Select **Add**.
+
+   Repeat the steps for each release you want to add to the schedule. The scheduled releases are listed in chronological order in the Desktop Wallet You can't edit the individual releases, but you can delete a release by selecting the Trash bin next to it.
+
+#. When the schedule is complete, select **Continue**. You can see the release schedule under **Transaction Details**, and you can generate the transaction.
+
+Generate the transaction
+========================
+
+There are two ways that you can generate the transaction:
+
+-  :ref:`Generate the transaction without signing<generate-scheduled-without-sign>`. This option enables you to export the transaction proposal without signing it. You don't need a Ledger or an internet connection to do this.
```

Comment:
```suggestion
-  :ref:`Generate the transaction without signing<generate-scheduled-without-sign>`. This option enables you to export the transaction proposal without signing it. You don't need a Ledger, but you do need to be connected to the internet to do this.
```
This is one of those places that needs to be reworded regarding internet connection for unsigned proposals.